### ISSUE TITLE
komodo dragon riding very busted

### ISSUE THREAD

**Issue by BrayDoesStuffYT** [Alex-the-666/AlexsMobs#1189]
im pretty sure when i ride a komodo they still have control. they are very slow when i ride them & they seem to only move sideways.

**Comment by gelatorob** [comment]
Yeah same here. I see that someone else had that same issue and that  issue was closed and the comment was (fix #1162)  so idk if it was acually fixed

**Comment by Willowflame** [comment]
I'm having the same issue with my leviathan where mob riding just doesn't work properly
